packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -134,10 +134,12 @@ export async function ensurePincode(): Promise<boolean> {
 
   if (pincodeType === PincodeType.PhoneAuth) {
     try {
-      await getPincodeWithBiometrics()
+      await getPincodeWithBiometry()
       return true
     } catch (error) {
-      Logger.warn(`${TAG}@ensurePincode`, `Retrieve PIN by biometrics error`, error)
+      Logger.error(`${TAG}@ensurePincode`, `Retrieve PIN by biometry error`, error)
```

Comment:
I saw Sentry captured an error about user cancelling this: https://sentry.io/organizations/valora-inc/issues/2956635589/?environment=org.celo.mobile.alfajores.dev&project=1250733&query=canceled&statsPeriod=14d
We should check that here and just warn for that case.

Reply:
yeah this was the issue i mentioned above, where the keychain lib throws an error for the user cancelled event and we were catching it in `retrieveStoredItem` and throwing a generic `throw new Error(ErrorMessages.KEYCHAIN_STORAGE_ERROR)`

i've attempted to exclude the user cancelled error from the genuine errors the same way as we exclude network errors, would be nice to see if you agree with the handling :)